src/app/(site)/signup/page.tsx

```diff
@@ -66,8 +69,14 @@ export default function signup() {
     },
   ];
 
-  const handleSubmit = (e: any) => {
+  const handleSubmit = async (e: React.FormEvent<HTMLFormElement>) => {
     e.preventDefault();
+    
+    if (!regData.name || !regData.email || !regData.password)
+			return createError('Vänligen fyll i alla fält');
```

Comment:
Here you are creating an error but never showing it in the UI. I recommed you to import "errorMessage" from useAuth and use it in the UI.

Example:
```tsx
...
export default function signup() {
  const { register, errorMessage, createError } = useAuth();
  
  ...

  return (
    <div className="registration sec-cont">
      <form onSubmit={handleSubmit}>
        <h2>Registrera dig</h2>
        {/* Use the error message */}
	{errorMessage && (
	<>{errorMessage}</>
	)}
        {inputs.map((input) => (
          <FormInput key={input.id} {...input} onChange={onChange} />
        ))}
        <button className="primary-btn">Registrera </button>
        <Link href="/" className="cancel-btn">
          Avbryt
        </Link>
      </form>
    </div>
  );
}

```